llvm/test/Transforms/LoopVectorize/check-no-vectorize.ll

```diff
@@ -0,0 +1,36 @@
+; This test asserts that we don't emit both
+; successful and unsuccessful message about vectorization.
+
+; RUN: opt -passes=loop-vectorize -debug -disable-output -pass-remarks-missed=loop-vectorize %s 2>&1 | FileCheck %s
+; CHECK-NOT: LV: We can vectorize this loop
+; CHECK: LV: Not vectorizing: Cannot prove legality
+
+target datalayout = "e-m:e-i8:8:32-i16:16:32-i64:64-i128:128-n32:64-S128-Fn32"
+target triple = "aarch64-unknown-linux-gnu"
```

Comment:
Does this test depend on the target triple? If so, it must go in the AArch64 directory. Though ideally it would be triple-independent.

Reply:
No, this does not depend on TT. I have done the change.